client.js

```diff
@@ -135,7 +136,8 @@ Discord.Client = class Client extends Discord.Client {
 			a[s.id] = {
 				id: s.sessionID,
 				sequence: s.sequence,
-				lastConnected: Date.now()
+				lastConnected: Date.now(),
+				shardCount: this.ws.shards.size
```

Comment:
this should be `this.ws.totalShards`

Reply:
Resolved by https://github.com/timotejroiko/discord.js-light/pull/41/commits/1e077a12fabf9d89d42de898bd0386defce5502d